explorer/pages/explore/[slug]/[...param].tsx

```diff
@@ -10,15 +10,24 @@ import { detail, listAll } from "../../../app/entity/api/service";
 import { getCurrentEntity } from "app/hooks/useCurrentEntity";
 import { DetailModel } from "../../../app/models/viewModels";
 import { Project } from "../../../app/views/Project/project";
+import { UUID_PARAM_INDEX } from "app/shared/constants";
 
 interface PageUrl extends ParsedUrlQuery {
   slug: string;
-  uuid: string;
+  param: string[];
```

Comment:
Would it be possible to refactor `param` to `params` to help indicate it is an array of path param values?